packages/components-react/projects/react-ssr-wrapper/tests/unit/specs/__snapshots__/react-ssr-wrapper.spec.tsx.snap

```diff
@@ -1268,8 +1268,11 @@ button::before {
   right: -4px;
   top: 6px;
   bottom: 6px;
+  transition: background-color var(--p-transition-duration, .24s) ease;
```

Comment:
We shouldn't output any hover styles no server side since they are unnecessary for the first split seconds of page load.  
I guess the `stripFocusAndHoverStyles` utility needs to be extended to handle both, the `build` and `build-prod` scenario.